.buildkite/pipelines/on_merge_unsupported_ftrs.yml

```diff
@@ -85,3 +85,5 @@ steps:
       automatic:
         - exit_status: '*'
           limit: 1
```

Comment:
We may want to put the catch-all `*` at the end of the list (after the `exit_status` 10) because apparently the order matters...

I found [this](https://forum.buildkite.community/t/ordering-of-automatic-retry-attributes/3178/3) Buildkite forum entry, but I haven't tried this myself.

From your experience, if we've tried this the different exit codes in the CI, does the order matter?

On the other hand, `scout_tests.yaml` seem to have the (supposedly) correct order.